### DISCUSSION TITLE
Anyone developing a Roku app for Stash?

### DISCUSSION THREAD

**Discussion by RRocks01** [stashapp/stash#3455] [Q&A]
I've been hoping to see a post pop up on this topic for some time so thought I'd just ask.  It seems like an app even an unsupported app for roku that could show the full stash interface would take this project to the moon.  To me it is the biggest roadblock preventing me from being able to really use it, well at least with the wife, which would be awesome.  I know I can cast it from my phone, or maybe use a web browser on a smart tv, but those options are clunky in comparison.  Crossing my fingers this happens some day.  Thanks All.

**Comment by jeremymeyers** [comment]
Developing adult-themed apps for roku is a risky proposition because its a lot of work and it could just be removed at any time.

**Comment by gsutoker** [comment]
@RRocks01 - I created an app that can be sideloaded to your Roku device.  Still in early testing phases.  But it should work! 
https://github.com/gsutoker/Stash-App---Roku
